run.py

```diff
@@ -182,20 +145,30 @@ def generate_topup_input(context):
 
     log = logging.getLogger('[flywheel/fsl-topup/generate_topup_input]')
```

Comment:
Yeah,
You can set the logger at the top of the script... or in the `if __name__ == '__main__'  section.